### DISCUSSION TITLE
swiss front theme need to integrate custom form popup

### DISCUSSION THREAD

**Discussion by vikeni** [breezefront/community#193] [1. Help] [answered]
Hello There,

I tried to show modal popup using many ways

1) Tried to integrate bootstrap.min.js but didn't work
2) Amasty custom form extension ( not sure if this supports or not )
3) using Magento_Ui/js/modal/modal but didn't work either.

Is there any way in which i can show custom form popup ?

Please help me on this.

**Comment by vovayatsyuk** [comment, marked as answer]
The correct way is to use the built-in modal: https://breezefront.com/docs/modal#initialization

**Reply by vikeni** [reply]
Thanks @vovayatsyuk it worked
